content/tutorials/node/hello-world.md

```diff
@@ -101,71 +101,63 @@ const rl = createInterface({
   output: process.stdout,
 });
 
-const produce = async () => {
-    // Connect the fluvio cluster;
-    await fluvio.connect();
+(async () => {
+  // Connect the fluvio cluster;
+  await fluvio.connect();
 
-    // Create a topic producer;
-    const producer = await fluvio.topicProducer("hello-fluvio");
-    console.log("Fluvio Producer created, waiting for input:\n\n>")
-    // Relay terminal input to fluvio topic producer;
-    rl.on("line", async (input) => {
-        await producer.send("line", input);
-    });
-};
+  // Create a topic producer;
+  const producer = await fluvio.topicProducer("hello-node");
+  const partition = 0;
+  console.log("Fluvio Producer created, waiting for input:\n\n>")
+  // Relay terminal input to fluvio topic producer;
+  rl.on("line", async (input) => {
+    await producer.sendRecord(input, partition);
+  });
+})();
 
-produce();
 ```
 
+### Writing the `consumer.ts` File
+
+Write the following code in your `consumer.ts` file.
+
 ##### This code performs the following actions:
 
-- _Import `@fluvio/client` and Node.js' `readline` modules;_
+- _Import `@fluvio/client` module;_
 - _Create a new Fluvio Client Instance;_
 - _Create a connection to a local Fluvio Cluster;_
-- _Create a new topic producer for `hello-fluvio`;_
-- _Listen for input typed into the terminal;_
-- _Send typed input to the fluvio cluster;_
-
-
-### Writing the `consumer.ts` File
+- _Create a new topic consumer for `hello-node`;_
+- _Listen for events sent by a topic producer;_
 
-Write the following code in your `consumer.ts` file.
 
 ```TypeScript
-import Fluvio, { Offset } from "@fluvio/client";
+import Fluvio, { OffsetFrom } from "@fluvio/client";
 
 // Create Fluvio Client Instance
 const fluvio = new Fluvio();
 
-const consume = async () => {
-    // Connect the fluvio cluster;
-    await fluvio.connect();
-
-    const partition = 0;
-
-    // Create Topic Consumer
-    const consumer = await fluvio.partitionConsumer("hello-fluvio", partition);
-    console.log("Fluvio Consumer created, listening for events:\n\n")
-    const stream = await consumer.createStream(Offset.FromBeginning());
-    
-    for await (const record of stream) {
-        const key = record.keyString();
-        const value = record.valueString();
-        console.log(`Received record: Key=${key}, value=${value}`);
+(async () => {
+  // Connect the fluvio cluster;
+  await fluvio.connect();
+
+  const partition = 0;
+
+  //   Create Topic Consumer
+  const consumer = await fluvio.partitionConsumer("hello-node", partition);
+  console.log("Fluvio Consumer created, listening for events:\n\n")
+  await consumer.stream(
+    {
+      index: 0,
+      from: OffsetFrom.Beginning,
+    },
+    async (msg: string) => {
+      console.log(`Received message: ${msg}`);
     }
-};
+  );
+})();
```

Comment:
Hmm looking at this, I think you may have undone some of my recent edits in your last rebase. This is supposed to stay how it was (e.g. `Offset`, not `OffsetFrom`, and `consumer.createStream` rather than `consumer.stream`)